pam_mysql.c

```diff
@@ -3087,8 +3130,55 @@ static void pam_mysql_saltify(pam_mysql_ctx_t *ctx, char *salt, const char *salt
   }
 #endif
 
+	if ((ctx->blowfish + ctx->sha512 + ctx->sha256 + ctx->md5) > 1) {
+		syslog(LOG_AUTHPRIV | LOG_ERR, "Only one of blowfish, sha512, sha256 or md5 should be specified. Falling back to the strongest of selected values.");
+	}
+
   q = salt;
-  if (ctx->md5) {
+
+#ifdef HAVE_BLOWFISH
+	if (ctx->blowfish) {
+		strcpy(salt, "$2a$");
+    q += sizeof("$2a$") - 1;
```

Comment:
Thanks for the fixes to whitespace elsewhere but you seem to also be introducing a few here and on lines 3153 and 3180. Could I get you to adjust them, please?

Reply:
Alien indentation style again.
The whitespace changes would be best put into a dedicated patch so that they can be reviewed separately.